molecule/common/requirements.yml

```diff
@@ -0,0 +1,6 @@
+---
+roles:
+  - src: git+https://github.com/infrawatch/functional-tests
+    name: functional_tests
+  - src: git+https://github.com/infrawatch/qdr-config-ansible-role
+    name: qdr-config-ansible-role
```

Comment:
I suggest installing this as qdr_config, as this is the name we'll expect to use when this moves to tripleo-ansible.